migrations/20200822184341-create-user-model.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.sequelize.query(`
+      --
+      -- ER/Studio Data Architect SQL Code Generation
+      -- Project :      BATS Logical Data Model.DM1
+      --
+      -- Date Created : Sunday, August 16, 2020 07:15:27
+      -- Target DBMS : PostgreSQL 9.x
+      --
+      
+      -- 
+      -- TABLE: batsuser 
+      --
+
+      CREATE TABLE batsuser(
+          user_uuid             uuid             NOT NULL,
+          firstname             varchar(50)      NOT NULL,
+          lastname              varchar(50)      NOT NULL,
+          email                 varchar(50),
```

Comment:
If we change the email column type to CITEXT (case-insensitive text), then we can also put a UNIQUE constraint on this column, and it will enforce case-insensitive unique email addresses.

Reply:
I see `email` listed as CITEXT in Bill's migration too, not sure why I put varchar... Thanks for the catch